### DISCUSSION TITLE
No more NotFoundException

### DISCUSSION THREAD

**Discussion by loicmathieu** [micronaut-projects/micronaut-core#11149] [Q&A] [answered]
Hi,
There is no more NotFoundException in recent Micronaut version, but it's still documented here: https://docs.micronaut.io/4.6.3/api/io/micronaut/http/exceptions/NotFoundException.html

**Comment by graemerocher** [comment]
that link returns NotFound 404 (the irony?)

**Reply by graemerocher** [reply]
are you referring to https://docs.micronaut.io/4.6.3/api/io/micronaut/http/server/exceptions/NotFoundException.html ? which was added in 4.6.0

**Reply by loicmathieu** [reply]
Yes so it's only the link in this section that redirect to the wrong URL https://docs.micronaut.io/4.6.4/guide/#builtInExceptionHandlers

**Reply by graemerocher** [reply, marked as answer]
https://github.com/micronaut-projects/micronaut-core/pull/11180
